tests/unit/scene/test_to_ipynb_converstion.py

```diff
@@ -0,0 +1,61 @@
+# Pyflow an open-source tool for modular visual programing in python
+# Copyright (C) 2021-2022 Bycelium <https://www.gnu.org/licenses/>
+
+"""Unit tests for the conversion to ipynb."""
+
+from typing import Dict, List, OrderedDict
+from pytest_mock import MockerFixture
+import pytest_check as check
+import json
+
+from pyflow.scene.to_ipynb_conversion import ipyg_to_ipynb
+from pyflow.scene.ipynb_conversion_constants import BLOCK_TYPE_TO_NAME
+
+
+class TestToIpynbConversion:
+
+    """Conversion to .ipynb"""
+
+    def test_empty_data(self, mocker: MockerFixture):
+        """should return an empty list of cells when the ipyg graph is empty."""
+        ipynb_data = ipyg_to_ipynb({"blocks": [], "edges": []})
+        check.equal("cells" in ipynb_data, True)
+        check.equal(type(ipynb_data["cells"]), list)
+        check.equal(len(ipynb_data["cells"]), 0)
+
+    def test_mnist_example(self, mocker: MockerFixture):
+        """should return the right number of cells in a right order on the mnist example."""
```

Comment:
I feel like the mnist file is too big for a unit test ...
Maybe take the "simple" .ipynb asset and add a branch in a graph ?

This makes a good integration test though !

Reply:
Done!